src/Types.php

```diff
@@ -26,6 +26,46 @@ class Types
     const TYPE_STRING = 'string';
 
     /**
+     * @var array
+     */
+    public static $phpToJson = [
+        'array' => self::TYPE_ARRAY,
+        'boolean' => self::TYPE_BOOLEAN,
+        'double' => self::TYPE_NUMBER,
+        'integer' => self::TYPE_INTEGER,
```

Comment:
I understand. That's a good idea. I'm just concerned about exposing those maps with the number/integer trick. This might be a source of bugs in the long run.